test/unit/adapters/http.js

```diff
@@ -1,21 +1,23 @@
+'use strict';
```

Comment:
Avoid formatting the whole file. Now I can't find what tests you changed or added.

Reply:
I reverted the changes to the formatting and the changes fixed some lint errors in the unit tests